framework/Models/PostProcessors/Validations/PhysicsGuidedCoverageMapping.py

```diff
@@ -0,0 +1,218 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+  Created on August 09, 2021
+
+  @author: dhuang
+
+  This class is for the algorithms of Physics-guided Coverage Mapping
+  It inherits from the PostProcessor directly
+  ##TODO: Recast it once the new PostProcesso API gets in place
+"""
+
+#External Modules------------------------------------------------------------------------------------
+import numpy as np
+import xarray as xr
+from scipy import stats
+#External Modules End--------------------------------------------------------------------------------
+
+#Internal Modules------------------------------------------------------------------------------------
+from utils import utils, randomUtils
+from .. import ValidationBase
+#Internal Modules End--------------------------------------------------------------------------------
+
+class PhysicsGuidedCoverageMapping(ValidationBase):
+  """
+    PhysicsGuidedCoverageMapping is a base class for validation problems
+    It represents the base class for most validation problems
```

Comment:
Please update the docstring to reflect the PhysicsGuidedCoverageMapping class

Reply:
Updated.